docs/examples/sentiment_analysis/index.rst

```diff
@@ -4,24 +4,22 @@ Sentiment Analysis
 .. container:: cards
 
    .. card::
-      :title: Fine-tuning LSTM-based Language Model
+      :title: Sentiment Analysis by Fine-tuning Word Language Model
       :link: sentiment_analysis.html
 
       See how to fine-tune a pre-trained language model to perform sentiment analysis on movie reviews.
 
    .. card::
-      :title: Training Structured Self-attentive Sentence Embedding
-      :link: self_attentive_sentence_embedding.html
-
-      See how to use GluonNLP to build more advanced model structure for extracting sentence
-      embeddings to predict Yelp review rating.
-
+      :title: Sentiment Analysis with KoBERT (Korean BERT) using Movie Review Data
+      :link: kobert_naver_movie.html
 
+      See how to use KoBERT to implement a sentiment analysis of a Korean popular movie review dataset.
 
 .. toctree::
    :hidden:
    :maxdepth: 2
 
    sentiment_analysis.ipynb
-   self_attentive_sentence_embedding.ipynb
+   kobert_naver_movie.ipynb
+
 
```

Comment:
Please keep the original `self_attentive_sentence_embedding.ipynb` contents and add your card alongside with the existing one.

Reply:
Done. Thanks!